src/popup/pages/Popups/AccountList.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <Modal
+    full-screen
+    class="connect"
+    data-cy="popup-aex2"
+  >
+    <TransactionInfo
+      :custom-title="$t('pages.connectConfirm.title')"
+      :sender="{ name: appName, address: app.host, url: app.url }"
+      :recipient="recipient"
+    />
+    <div
+      class="subtitle"
+      data-cy="aepp"
+    >
+      <span class="app-name">{{ appName }}</span>
+      ({{ app.host }}) {{ $t('pages.accountListConfirm.websiteRequestconnect') }}
+    </div>
+
+    <div class="permissions">
+      <template v-if="access.includes(POPUP_CONNECT_ADDRESS_PERMISSION)">
+        <span class="title">
+          <CheckMark class="icon" /> {{ $t('common.address') }}
+        </span>
+        <span class="description">
+          {{ $t('pages.accountListConfirm.addressesRequest') }}
+        </span>
+        <span class="description message">
+          {{ $t('pages.accountListConfirm.message') }}
+        </span>
```

Comment:
I suggest to replace the spans with `<p>`. This way you wan't need to set `display: block` in the CSS definitions block.

Reply:
I have tried maintaining this structure as other popup files. Maintains readability.

Reply:
> I have tried maintaining this structure as other popup files
You mean you just copy+pasted the code from `Connect.vue`? :) I don't think this is a good way of solving the problem of code duplication?

Reply:
Yes, It's a duplicate of `Connect.vue`. Imagine supporting 3 popup types and having just 2 files under `/Popups` directory. Someone reading fresh would get confused. Either the solution is to combine all 3 into one or maintain 3 files. But will not maintain 2 files where one file just represents two popup types.

Reply:
I don't like code duplication, but if you insist on leaving it as it is I'm OK with it. Maybe we'll refactor this in the future.